tracer/src/Datadog.Tracer.Native/dd_profiler_constants.h

```diff
@@ -50,6 +50,7 @@ const shared::WSTRING skip_assembly_prefixes[]{
     WStr("System.Text"),
     WStr("System.Threading"),
     WStr("System.Xml"),
+    WStr("testhost")
```

Comment:
@tonyredondo will this cause issues for CI app? 🤔

Reply:
I have removed it. Everything works fine.